src/App.js

```diff
@@ -573,6 +581,31 @@ export default function App() {
                 5v5
               </ToggleButton>
             </ToggleButtonGroup>
+            <br />
+            <strong>Teammates: </strong>
```

Comment:
I would use another "title" for this filter.
You don't really filter on "team mates" but rather on "team composition" because with your current implementation you also have to toggle your own class in the list.

Reply:
changed